base/bitarray.jl

```diff
@@ -1503,16 +1503,8 @@ function findnextnot(B::BitArray, start::Integer)
 end
 findfirstnot(B::BitArray) = findnextnot(B,1)
 
-# returns the index of the first matching element
-function findnext(B::BitArray, v, start::Integer)
```

Comment:
I suspect we should keep these for efficiency, but change the signature to `findnext(pred::EqualTo, B::BitArray, start)`. Then inside the function you need to do `v = pred.x`.

Reply:
Isn't that mostly covered by [this method](https://github.com/JuliaLang/julia/blob/8e7ae9f96fa2eee7316a7a4a205cede4d22ae917/base/bitarray.jl#L1515-L1525)? The extra code is `O(1)`.

Reply:
Yeah, maybe. So you think there was no reason for the existence of these methods in the first place?